_layouts/meltano_plugin.html

```diff
@@ -326,21 +295,47 @@ <h3 id="next-steps">Next steps</h3>
 
 <h2 id="capabilities">Capabilities</h2>
     
-{% if variant.capabilities.size == 0 %}
+{% if plugin.capabilities.size == 0 %}
```

Comment:
Can we only show this for taps? SQLFluff and other utilities without capabilities are fine.

Reply:
Resolved via 772fa1ba19c2ce878d480ce03c63a628e39a3fa5